### PR TITLE
Add dash 2 cask

### PR DESCRIPTION
**Pull request by nicday**

### CODE REVIEW

File: Casks/dash-2.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cask :v1 => 'dash-2' do
```
Comment (vitorgalvao) [inline Casks/dash-2.rb]: No `-` before numbers.

Comment (nicday) [inline Casks/dash-2.rb]: No worries, `-` removed

File: Casks/dash2.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+cask :v1 => 'dash2' do
+  version '2.2.5'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/dash2.rb]: `sha256` also needs to be updated. You can find what it is by running `shasum -a 256` on the downloaded file.

**Comment by fanquake** [comment]
@nicday Could you make the required changes and squash your commits?

**Pull request merged**